tests/countries/test_united_states.py

```diff
@@ -1070,6 +1083,21 @@ def test_lincolns_birthday(self):
             self.assertNoNonObservedHolidayName(
                 f"{name} (observed)", UnitedStates(subdiv=subdiv, observed=False), obs_dt
             )
+
+        obs_dt = (
```

Comment:
IMO we should probably standardize US test files to use `dts` and `obs_dts` like other files in another PR